apps/arena/lib/arena/game_updater.ex

```diff
@@ -359,14 +378,18 @@ defmodule Arena.GameUpdater do
     {:noreply, state}
   end
 
-  def handle_info(:zone_shrink, %{game_state: %{zone: %{shrinking: true}}} = state) do
-    Process.send_after(self(), :zone_shrink, state.game_config.game.zone_shrink_interval)
-    radius = max(state.game_state.zone.radius - state.game_config.game.zone_shrink_radius_by, 0.0)
-    state = put_in(state, [:game_state, :zone, :radius], radius)
+  def handle_info(:zone_shrink, %{game_state: %{zone: %{shrinking: false}}} = state) do
     {:noreply, state}
   end
 
-  def handle_info(:zone_shrink, %{game_state: %{zone: %{shrinking: false}}} = state) do
+  def handle_info(:zone_shrink, %{game_state: %{zone: %{enabled: false}}} = state) do
```

Comment:
I think we have to include a :zone_shrink call here, because:
1. We toggle the zone to false
2. This pattern match got executed
3. we toggle the zone to true 
4. We'll never see the zone again since it doesn't do anything